grails-app/views/projectDocument/_index.gsp

```diff
@@ -0,0 +1,26 @@
+<g:javascript>
+    const user = {
+        <g:if test="${session?.user}">
+            displayName: "${session.user.displayName}",
+            email: "${session.user.emailAddress}",
+            userName: "${session.user.userName}"
+        </g:if>
+    };
+    const component = {
+      user: user,
+      projectKey:'${issue.projectKey}',
+      attachedDocumentsUrl: "${createLink(uri: '/api/files-helper/attached-documents', method: 'GET')}",
+      attachDocumentsUrl: "${createLink(uri: '/api/files-helper/attach-document', method: 'POST')}",
+      issue: '${issue.requestDate}',
+      projectUrl: "${createLink(controller: 'project', action: 'getProject')}",
+      roles: '${session.roles}'.replace(/\s+/g, '').replace("[", "").replace("]", "").split(","),
```

Comment:
We should not be referencing session in these GSP pages. All variable content should be queried via a service call.